lib/rspec/sidekiq/matchers/base.rb

```diff
@@ -227,13 +285,34 @@ def failure_message
 
               message << job_messages.join("\n")
             end
+          else
+            message << "but enqueued 0 jobs"
           end
 
           message.join("\n")
         end
 
+        def common_message
+          "#{prefix_message} #{count} #{klass} #{expected_count.last == 1 ? "job" : "jobs"}"
+        end
+
+        def prefix_message
+          raise NotImplementedError
+        end
+
+        def count
```

Comment:
🎨 Nitpick but could we rename this to `count_message` or `job_count_message` (or something similar) so it's clear it's about crafting a message?